Xamarin.Forms.ControlGallery.Tizen/Xamarin.Forms.ControlGallery.Tizen.csproj

```diff
@@ -1,18 +1,13 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Tizen.NET.Sdk/1.0.8">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>tizen40</TargetFramework>
+    <TargetFramework>tizen50</TargetFramework>
```

Comment:
Not sure we want to update this, do we need to update tizen ?

Reply:
I was not able to run project with tizen40. also as tizen5 exist and we support it, I decided to update the csproj.

Reply:
@rookiejava what's your opinion ?

Reply:
I think we need to revert this